src/drupal8/application/overlay/phpunit.xml.twig

```diff
@@ -22,8 +22,8 @@
         <!-- Example SIMPLETEST_BASE_URL value: http://localhost -->
         <env name="SIMPLETEST_BASE_URL" value="https://{{ @('hostname') }}/"/>
         <!-- Example SIMPLETEST_DB value: mysql://username:password@localhost/databasename#table_prefix -->
-        <env name="SIMPLETEST_DB" value="sqlite://ignored//tmp/test.sqlite"/>
+        <env name="SIMPLETEST_DB" value="sqlite://localhost//dev/shm/test.sqlite"/>
```

Comment:
/dev/shm in docker is only 64mb by default, will that be a problem?

Reply:
It seems to work okay :+1:

Reply:
to me it's not whether it works ok now, it's that the /dev/shm is small and shared between different purposes. If the db is loaded on a project with lots of data, it would interfere with the other purposes. That's why I'm thinking a tmpfs volume

Reply:
the shm is adjustable too also in docker-compose.yml tho